code/modules/projectiles/projectile/beams.dm

```diff
@@ -12,6 +12,7 @@
 	hitscan = 1
 	invisibility = 101	//beam projectiles are invisible as they are rendered by the effect engine
 	style_damage = 30 //hitscan, light speed projectiles? Be glad its easier to dodge than a revolver.
+	recoil = 1 // Even less than self-propelled bullets
```

Comment:
Recoil on lasers? C'mon.

Reply:
Mathematically a thing, even if insignificant.

This amount of recoil is small enough to not be noticable, but enough to allow us to implement gunmods that can make it _very_ noticable. Thunder damage mod comes to mind.